setup/Core/Utilities/HookGenerator.cs

```diff
@@ -271,6 +280,8 @@ public bool GenerateFor(TypeDefinition hookType, TypeDefinition hookILType, Meth
                 name = nameTmp;
             }
 
+            bool obsolete = method.HasCustomAttribute("System.ObsoleteAttribute"); // Do we need to check if error is true?
```

Comment:
No, it's generally fine for mods to keep calling old forwarder methods for longer than it is to hook them